lib/lotus/commands/new/app.rb

```diff
@@ -104,7 +104,7 @@ def upcase_app_name
         end
 
         def classified_app_name
-          Utils::String.new(app_name).classify
+          Utils::String.new(app_name).classify.tr('::', '')
```

Comment:
I thought about this. But we need to replace this method to utils repository for escape DRY. Also I thought about `Lotus::Generators::Generatable` but it incorrect place for this method too.